drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -209,6 +209,30 @@
     opacity: 0.80;
 }
 
+
+/* Horizontal scroll for item bank on mobile */
+.chromeless .xblock--drag-and-drop .target {
+    display: block;
+    overflow: auto;
+}
```

Comment:
Please remove these styles, these will be added in a separate task. This task is only about making the item bank scrollable, not the drop target image.